### PR TITLE
[plplot] no absolute paths

### PR DESCRIPTION
**Pull request by autoantwort**
For #20469

### CODE REVIEW

**Review by JonLiu1993 — APPROVED** [review]

File: ports/plplot/portfile.cmake [hunk 1]
```diff
@@ -71,6 +71,10 @@ endif()
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/share")
 
+if(NOT VCPKG_TARGET_IS_WINDOWS)
+    vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/share/plplot/examples/plplot-test.sh" "${CURRENT_BUILDTREES_DIR}" "")
```
Comment (BillyONeal) [inline ports/plplot/portfile.cmake:75]: Should we just delete the 'examples' directory?

Comment (autoantwort) [reply ports/plplot/portfile.cmake:75]: I don't care. We can also delete it.

**Comment by JonLiu1993** [comment]
@autoantwort ,This is the error log:
````
-- Building x64-linux-rel
-- Static build: Removing the full bin directory.
CMake Error at scripts/cmake/vcpkg_replace_string.cmake:12 (file):
  file failed to open for reading (No such file or directory):

    /mnt/vcpkg-ci/packages/plplot_x64-linux/share/plplot/examples/plplot-test.sh
````

**Review by JonLiu1993 — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for the fixes and for dealing with my inane CR comments!

**Pull request merged**